### DISCUSSION TITLE
serenity.proxy.sslProxy and serenity.proxy.sslProxyPort not working for serenity

### DISCUSSION THREAD

**Discussion by jpramirezlo** [serenity-bdd/serenity-core#2595] [Q&A] [answered]
I would like to know that I am performing wrong since for me those two serenity properties do not work, I have the following configuration in the serenity.properties file;

serenity.proxy.sslProxy = 127.0.0.1
serenity.proxy.sslProxyPort = 8080
webdriver.driver = firefox
webdriver.gecko.driver = src / test / resources / driver / geckodriver.exe

Estoy usando la versión de serenity: 2.0.84

**Comment by wakaleo** [comment, marked as answer]
You can use the serenity.proxy.* properties defined in https://javadoc.io/doc/net.serenity-bdd/serenity-model/latest/net/thucydides/core/ThucydidesSystemProperty.html in version 3.2.0.
